### PR TITLE
Add `operation not found` exception

### PR DESCRIPTION
**Pull request by extempl**

### CODE REVIEW

File: src/execute.js [hunk 1]
```diff
@@ -219,7 +236,7 @@ export function baseUrl({spec, scheme, contextUrl = ''}) {
 // Add security values, to operations - that declare their need on them
 export function applySecurities({request, securities = {}, operation = {}, spec}) {
   const result = assign({}, request)
-  const {authorized = {}, specSecurity = {}} = securities
+  const {authorized = {}, specSecurity = []} = securities
```
Comment (shockey) [inline src/execute.js]: Why was this changed?

File: src/execute.js [hunk 2]
```diff
@@ -48,6 +54,10 @@ export function execute({
 
   const request = self.buildRequest({spec, operationId, parameters, securities, ...extras})
 
+  if (request.error) {
```
Comment (shockey) [inline src/execute.js]: Why are we throwing in `execute`, via the `request.error` switch, instead of throwing the error directly in `buildRequest`?

File: src/execute.js (hunk 1 above)
Comment (extempl) [inline src/execute.js]: Because of variable type. When `{}` there will be exception  on `forEach` it: 'forEach is not a function'
    const {authorized = {}, specSecurity = []} = securities    
    const security = operation.security || specSecurity
  
    security.forEach

File: src/execute.js (hunk 2 above)
Comment (extempl) [inline src/execute.js]: Good point. That was done due to approach I used before. Updated

File: src/execute.js (hunk 1 above)
Comment (shockey) [inline src/execute.js]: aha. Good catch, thanks!

**Comment by extempl** [comment]
@shockey Test is added.
